chatterbot/storage/django_storage.py

```diff
@@ -138,6 +138,21 @@ def remove(self, statement_text):
         responses.delete()
         statements.delete()
 
+    def get_latest_response(self, conversation_id):
+        """
+        Returns the latest response in a conversation if it exists.
+        Returns None if a matching conversation cannot be found.
+        """
+        from django.apps import apps
```

Comment:
Importing third party packages at the top of a file sometimes causes unexpected errors depending on how users install ChatterBot. The best way (that I have found) to avoid this is to protect the imports by placing them inside of a method as this prevents them from being imported when the file is loaded. I'm actually not certain why some people encounter the import errors but it seems that it may be the result of less conventional (but still somewhat valid) installation methods such as installing directly through `setup.py` commands.